docpages/03_example_programs.md

```diff
@@ -1656,6 +1656,7 @@ The evaluated code will run within its own thread, so can execute for as long as
 This code operates by outputting your provided code to be evaluated into a simple boilerplate program which can be compiled to a
 shared object library (.so file). This .so file is then compiled with g++, using the `-shared` and `-fPIC` flags. If the program can be successfully compiled, it is then loaded using `dlopen()`, and the symbol `so_exec()` searched for within it, and called. This `so_exec()` function will contain the body of the code given to the eval command. Once this has been called and it has returned,
 the `dlclose()` function is called to unload the library, and finally any temporary files (such as the .so file and its corresponding .cpp file) are cleaned up.
+Docker is definitely recommended if you code on Windows/Mac OS, because docker desktop still uses a linux VM, so your code can easily use `.so` file and your code runs the same on your vps (if it also uses Linux distro)
```

Comment:
@braindigitalis Can you give your opinion on this?

Reply:
See down